packages/scrollregion/src/ScrollRegionContainer.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useScrollRegion, IUseScrollRegion } from './useScrollRegion';
+
+export interface IScrollRegionContainerProps extends IUseScrollRegion {
+  /** A render prop function which receives the tab index */
+  render?: (tabIndex: number | undefined) => React.ReactNode;
+  /** A children render prop function which receives the tab index */
+  children?: (tabIndex: number | undefined) => React.ReactNode;
```

Comment:
Do we know why the props aren't showing in storybook?

Reply:
Ah yes, I had forgotten to pass the container interface to the `React.FC` generic. It should be showing the prop & prop descriptions now 👍🏽 .